### DISCUSSION TITLE
How to import historical transactions?

### DISCUSSION THREAD

**Discussion by chansters** [ghostfolio/ghostfolio#915] [Q&A] [answered]
I'm self-hosting Ghostfolio (it looks great and super easy to setup) but I can't figure out how to important past transactions?

**Comment by dtslvr** [comment, marked as answer]
Hello @chansters

You can find the import functionality for `*.csv` and `*.json` files here:
<img width="1017" alt="ghostfolio-screenshot-import" src="https://user-images.githubusercontent.com/4159106/168224698-7270a6a6-87d0-4f2f-985e-75645638332d.png">

These are examples of the expected format: [csv](https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.csv) and [json](https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.json) (format of exported data).

**Reply by chansters** [reply]
Thank you! I didn't know how I missed that!
